instructor/templates/instructor/assignments.html

```diff
@@ -2,12 +2,10 @@
 
 {% block content %}
     <div class='scb_s_dashboard_sidebar'>
-        <h1 class="scb_s_dashboard_sidebar_title">Your Courses:
-            <div class="scb_s_dashboard_sidebar_name scb_s_dashboard_button_wrapper"></div>
-        </h1>
+        <h1 class="scb_s_dashboard_sidebar_title">Your Assignments: </h1>
 
-        <a class='scb_s_navigation_button scb_s_dashboard_new_assignment_button'
-           href="{% url "common_create_new_assignment" %}">Create a New Assignment</a>
+        <a class='scb_ab_s_navigation_button scb_s_dashboard_new_assignment_button'
+           href="{% url "common_create_new_assignment" %}">Create a New Assignment&nbsp; +</a>
```

Comment:
Because this is how Shloka had it in her template, and it seemed pretty straight forward. How would add space within a string using css?